content/get-involved/collab-summit.pt.md

```diff
@@ -0,0 +1,18 @@
+---
+title: 'collab-summit'
+displayTitle: 'Cimeira de Colaboração'
+description: 'Cimeira de Colaboração | Node.js'
+category: get-involved
+---
+
+A Cimeira de Colaboração é uma conferência em pessoa para ajuntar os atuais e potenciais colaboradores para discutir a Node.js com colaboração, educação, e partilha de conhecimento animada. Os comités e grupos de trabalho ajuntam-se duas vezes por ano para tomar decisões importantes enquanto também são capazes de trabalhar em entusiasmantes esforços que eles querem prosseguir em pessoa.
```

Comment:
Cimeira? Talvez uma palavra mais genérica seria "Encontro", "Evento"?

Reply:
:gb:  @ovflowd I believe the word `Cimeira` works well due to its formal sound.
But if it sounds too strange or weird I will update it to `Encontro dos Colaboradores`.

:gb: For now I believe we could pass this because it will most likely be updated and 
corrected soon in the next pull requests over the main repository.

:portugal: @ovflowd Eu acredito que a palavra `Cimeira` encaixa-se bem devido ao
seu aspeto formal. Mas se ela soa formal demais ou estranho eu irei atualizá-la para
`Encontro dos Colaboradores`.

:portugal: Por agora acredito que podemos deixar passar esta porque muito provavelmente
será atualizado ou corrigido nos próximos pedidos de atualização do repositório principal.